config/roles.yml

```diff
@@ -284,13 +284,26 @@ ml_query_assistant_access:
 ml_read_access:
```

Comment:
Will it cause any BWC issue if user upgrade to 2.13 from an old cluster?

Reply:
Will double check this with security team

Reply:
It would cause bwc issues if it were a change in the role name. It is ok to change the permissions in a role. Cluster admins can either use securityadmin or a [new security api being introduced in 2.13](https://github.com/opensearch-project/documentation-website/pull/6634) to source changes to default roles into the security index.

Reply:
FYI changes to the role name are problematic because the role name is used in [`roles_mappings.yml`](https://opensearch.org/docs/latest/security/configuration/yaml/#roles_mappingyml) to map users to a role. If the role name changes, but the corresponding roles_mapping entry does not change, then those users would effectively lose access to functionality that they were able to previously use.

Reply:
Similarly, some plugins utilize roles injection for scheduled jobs to do permissions check on actions for scheduled jobs. If a role name changes, there is a risk that there are orphaned references to roles in disparate system indexes across the plugin ecosystem.

Reply:
Thanks for answering!